master.jinja2

```diff
@@ -4,15 +4,16 @@
 {% set deploy_timeout = deploy_timeout|default(15) %}
 {% set boot_timeout = boot_timeout|default(5) %}
 {% set install_fastboot_timeout = install_fastboot_timeout|default(10) %}
+{% set fastboot_deploy_timeout = fastboot_deploy_timeout|default(0) %}
 {% set target_deploy_timeout = target_deploy_timeout|default(40) %}
```

Comment:
Could you please add one comment here to say that target_deploy_timeout might be used for the deploy "to: downloads" action when DEPLOY_TARGET is "downloads", and no need to specify fastboot_deploy_timeout when DEPLOY_TARGET is not "downloads"?
to avoid confusion for the future users.

Reply:
I added a comment in the master.jinja2 file.

Reply:
Thanks for the update!